esx_service/vmdk_ops_test.py

```diff
@@ -885,22 +878,29 @@ def test_vmdkops_on_tenant_vm(self):
                                          vm_list=[self.vm1_name]) 
         self.assertEqual(None, error_info)
 
-        # run create command, which should succeed since we have DEFAULT privilege set
+        # run create command, which should fail since "default_datastore" is not set for tenant1
```

Comment:
why create() should fail without default_datastore ? I thought we wanted to keep the current behavior (i.e. use VM datastore) ? If this is not the case then we need to make setting default_datastore mandatory on tenant creation